### ISSUE TITLE
Navbar isn't resizing correctly

### ISSUE THREAD

**Issue by noxify** [mrcrmn/docc#16]
Hi,

it seems that the header navbar isn't resizing correctly.

## Steps to reproduce

1. Open the `gridsome.config.js`

2. update the `nav` property and replace it with 
```
nav: {
  links: [
    { path: '/', title: 'Home' },
    { path: '/docs/', title: 'Docs' }
  ]
},
```

3. Run `gridsome develop`

## Current Result
![image](https://user-images.githubusercontent.com/521777/80962376-3994e600-8e0d-11ea-94e8-0d0642c0107e.png)

## Expected Result

1. The Searchbar is resized (and smaller)
2. There is no "line break"

**Comment by infotexture** [comment]
Looks like this was fixed with #3, which was opened 2 months ago but hasn't been merged yet.

**Comment by noxify** [comment]
Will be solved with #3 - I close this, since there is already a fix.
